src/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -24,9 +29,29 @@ class Breadcrumbs extends Component {
         'aria-current': 'page',
       }),
     );
-    return children.map((child, i) => <li key={i}>{child}</li>);
+    return children.map((child, i) => (
+      <li key={i}>{React.cloneElement(child, { 'data-index': i })}</li>
+    ));
   };
 
+  handleBreadcrumbLinkClick(e) {
+    if (!this.props.disableAnalytics) {
+      // If it's a link being clicked, dispatch an analytics event
+      if (e.target?.tagName === 'A') {
+        dispatchAnalyticsEvent({
+          componentName: 'Breadcrumbs',
+          action: 'linkClick',
+          details: {
+            clickLabel: e.target.innerText.trim(),
+            clickLevel: parseInt(e.target.dataset.index, 10) + 1,
```

Comment:
Is the `data-index` thing done because `key` doesn't show up in the DOM? I think the answer is yes, and if so, can you add a short 1 line comment saying this?